assets/services/parachute/parachute.py

```diff
@@ -0,0 +1,182 @@
+import datetime
```

Comment:
Could we have bit of comment on the top explaining what this is for. Can point to the protocol doc. Ditto for the following one.